File: 3533-snake-in-matrix/snake-in-matrix.cpp
class Solution {
public:
    int finalPositionOfSnake(int n, vector<string>& commands) {
        int k = 0, j = 0;
        for (int i = 0; i < commands.size(); i++) {
            if (commands[i] == "RIGHT") j+=1;
            else if (commands[i] == "LEFT") j-= 1;
            else if (commands[i] == "UP") k -= 1;
            else k += 1;
        }
        return k*n + j;
    }
};

File: 3533-snake-in-matrix/snake-in-matrix.py
class Solution:
    def finalPositionOfSnake(self, n: int, commands: List[str]) -> int:
        x = 0
        y = 0
        for item in commands:
            if item == "RIGHT": y += 1
            elif item == "LEFT": y -= 1
            elif item == "UP": x -= 1
            else: x += 1

        return x*n + y